frontend/src/Search/SearchFooterConnector.js

```diff
@@ -18,13 +23,49 @@ function createMapStateToProps() {
         searchOffset: defaultSearchOffset
       } = releases.defaults;
 
+      const { params } = parseUrl(location.search);
+      const defaults = {};
+
+      if (params.query && !defaultSearchQuery) {
+        defaults.searchQuery = params.query;
+      }
+
+      if (params.indexers && !defaultIndexerIds.length) {
+        defaults.searchIndexerIds = params.indexers.split(',').filter(Boolean).map((i) => Number(i));
+      }
+
+      if (params.categories && !defaultCategories.length) {
+        const categoryIds = indexerCategories.items.flatMap((item) => {
+          const subCategoryIds = item.subCategories ? item.subCategories.map((sc) => sc.id) : [];
+          return [item.id, ...subCategoryIds];
+        });
+
+        defaults.searchCategories = params.categories
+          .split(',')
+          .filter((c) => categoryIds.includes(Number(c)))
+          .map((c) => Number(c));
+      }
+
+      if (params.type && defaultSearchType === 'search' && SEARCH_TYPES.includes(params.type)) {
+        defaults.searchType = params.type;
+      }
+
+      if (params.limit && defaultSearchLimit === 100 && !isNaN(params.limit)) {
```

Comment:
It does work because it converts the value to a number before checking. See in [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN#return_value](mdn docs).